Cargo.toml

```diff
@@ -9,76 +9,121 @@ authors = [
 	"Rajesh Malviya <rajveer0malviya@gmail.com>",
 ]
 edition = "2021"
-description = "WebGPU native implementation on gfx-hal"
+description = "Native WebGPU implementation based on wgpu-core"
 homepage = "https://github.com/gfx-rs/wgpu-native"
 repository = "https://github.com/gfx-rs/wgpu-native"
 keywords = ["graphics"]
 license = "MIT OR Apache-2.0"
 
+[workspace]
+resolver = "2"
+
+[workspace.dependencies.wgc]
+package = "wgpu-core"
+git = "https://github.com/gfx-rs/wgpu"
+rev = "484457d95993b00b91905fae0e539a093423cc28"
+version = "0.19"
+
+[workspace.dependencies.wgt]
+package = "wgpu-types"
+git = "https://github.com/gfx-rs/wgpu"
+rev = "484457d95993b00b91905fae0e539a093423cc28"
+version = "0.19"
+
+[workspace.dependencies.naga]
+package = "naga"
+git = "https://github.com/gfx-rs/wgpu"
+rev = "484457d95993b00b91905fae0e539a093423cc28"
+version = "0.19"
+
 [lib]
-crate-type = ["lib", "cdylib", "staticlib"]
+crate-type = ["cdylib", "staticlib"]
```

Comment:
intentional, don't think we want anyone to use `wgpu-native` as a rust dependency.